### PR TITLE
testing/elasticsearch: new abuild

### PR DESCRIPTION
**Pull request by jirutka**
https://www.elastic.co/products/elasticsearch
Open Source, Distributed, RESTful Search Engine

### CODE REVIEW

File: testing/elasticsearch/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	$pkgname.confd
+	"
+
+_modules="lang-expression lang-groovy"
+for _mod in $_modules; do
+	subpackages="$subpackages $pkgname-$_mod:_${_mod//-/_}"
+	eval "_${_mod//-/_}() { _builtin_module $_mod; }"
+done
+
+_builddir="$srcdir/$pkgname-$pkgver"
```
Comment (Barthalion) [inline testing/elasticsearch/APKBUILD]: I think it's being set by default as "$builddir" now. @ncopa, could you confirm?

**Comment by ncopa** [comment]
fixed a whitespace damage and pushed. thanks!

File: testing/elasticsearch/APKBUILD (hunk 1 above)
Comment (ncopa) [inline testing/elasticsearch/APKBUILD]: yes, abuild use "$builddir" now, due to the `default_prepare` func in abuild.

Comment (jirutka) [inline testing/elasticsearch/APKBUILD]: Thanks for notice, I didn’t know about it. Could you please add it to wiki http://wiki.alpinelinux.org/wiki/APKBUILD_Reference#Variables?

**Pull request closed without merging**